assets/src/scss/blocks/PostsList/PostsListEditor.scss

```diff
@@ -0,0 +1,30 @@
+.posts-list {
+  .carousel-controls {
```

Comment:
I think we can move this class within `&.is-custom-layout-list` and get rid of the `!important` statement.

Reply:
In the editor we actually always want to hide the carousel controls, not only for the list style